src/it/scala/ch/epfl/bluebrain/nexus/perf/AddAttachmentSimulation.scala

```diff
@@ -0,0 +1,79 @@
+package ch.epfl.bluebrain.nexus.perf
+import java.net.URLEncoder
+
+import ammonite.ops._
+import ch.epfl.bluebrain.nexus.perf.config.Settings
+import com.typesafe.config.ConfigFactory
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+
+import scala.util.Random
+
+class AddAttachmentSimulation extends Simulation {
+
+  val config = new Settings(ConfigFactory.parseResources("perf-tests.conf").resolve()).appConfig
+
+  val schemas = List(
```

Comment:
these schemas seem to be the same everywhere, could you externalize the map to an object?